doc/sphinxext/gallery_generator.py

```diff
@@ -398,6 +400,9 @@ def main(app):
                 img_file=ex.pngfilename,
                 api_name=ex.apiname,
             )
+            if backend == "matplotlib":
+                with open(op.join(target_dir, ex.pyfilename), "a") as f:
+                    f.write("plt.show()")
```

Comment:
I used this workaround to remove matplotlib popups. Is this okay?

Reply:
I think it would be better to use something like [this](https://github.com/arviz-devs/arviz/blob/dfd353d69a9b9eed4a871c5f3d2d37255da996ed/doc/sphinxext/gallery_generator.py#L307) we tried this with bokeh initially. The generated website will be the same, but people having cloned the repo will be able to run `python example.py` and get a plot as plt.show() will be in the file

Reply:
I have done the changes.